packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -147,29 +147,19 @@ export function rscHmrPlugin(): Plugin {
         );
       }
     },
-    handleHotUpdate({ file }) {
+    handleHotUpdate({ file, modules, server }) {
+      const module = server.moduleGraph.getModuleById(file);
+      if (module?.file?.endsWith('.module.css')) {
+        module.importers.forEach((importer) => {
+          handleModuleUpdate(importer.file!, server);
+        });
+      }
```

Comment:
This is just to track the css module files! I wonder if we can have a better code for this. I just did it because it's part of the fix.